Fallback to `@TERMUX_PREFIX_CLASSICAL@/tmp` if env `XDG_RUNTIME_DIR` is not set.

--- a/src/util/anon_file.c
+++ b/src/util/anon_file.c
@@ -136,6 +136,9 @@
    char *name;
 
    path = getenv("XDG_RUNTIME_DIR");
+   if (!path) {
+      path = "@TERMUX_PREFIX_CLASSICAL@/tmp";
+   }
    if (!path) {
       errno = ENOENT;
       return -1;
